e2e-tests/playwright/utils/UIhelper.ts

```diff
@@ -90,7 +90,21 @@ export class UIhelper {
   }
 
   async openSidebar(navBarText: string) {
-    await this.page.locator(`nav a`).filter({ hasText: navBarText }).click();
+    await this.page.click(`nav a:has-text("${navBarText}")`);
+  }
+
+  async verifyTableHeaders(expectedHeaders: string[]) {
+    for (const header of expectedHeaders) {
+      const headerLocator = this.page.locator(
+        `${UIhelperPO.MuiTable} thead th span div >> text="${header}"`,
```

Comment:
Some th elements might not contain a span div. Can we validate the text in any descendant of th instead?